com.beatwaves.responsible/Runtime/Bdd/Bdd.Unity.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using NUnit.Framework.Internal.Builders;
+using static Responsible.Bdd.Keywords;
+
+namespace Responsible.Bdd
+{
+	public abstract class BddTest
```

Comment:
This class is required because AFAIK, NUnit doesn't really support set ups, tear downs etc, if we make it execute a method in any other class. Some discussion [here](https://github.com/nunit/nunit/issues/616). Also, Unity makes this even more complicated with `IEnumerableTestMethodCommand` being internal.